corehq/apps/formplayer_api/management/commands/clear_formplayer_dbs.py

```diff
@@ -0,0 +1,107 @@
+        """
+    )
+
+    def create_parser(self, *args, **kwargs):
+        # required to get nice output from `--help`
+        parser = super(Command, self).create_parser(*args, **kwargs)
+        parser.formatter_class = RawTextHelpFormatter
+        return parser
+
+    def add_arguments(self, parser):
+        parser.add_argument('--from-csv',
+                            help='Path to CSV file. Columns "domain, username, as_user". When this is supplied '
+                                 'only users in this file will be synced.')
+
+        parser.add_argument('--domains', nargs='+', help='Match users in these domains.')
+        parser.add_argument('--last-synced-hours', action=validate_integer(gt=0, lt=673), default=48,
+                            help='Match users who have synced within the given window. '
+                                 'Defaults to 48 hours. Max = 472 (4 week).')
+        parser.add_argument('--not-synced-hours', action=validate_integer(gt=0, lt=169),
+                            help='Exclude users who have synced within the given window. '
+                                 'Max = 168 (1 week).')
+
+        parser.add_argument('--limit', action=validate_integer(gt=0), help='Limit the number of users matched.')
+        parser.add_argument('--dry-run', action='store_true', help='Only print the list of users.')
+        parser.add_argument('--dry-run-count', action='store_true', help='Only print the count of matched users.')
+
+    def handle(self,
+               from_csv=None,
+               domains=None,
+               last_synced_hours=None,
+               not_synced_hours=None,
+               limit=None,
+               **options
+               ):
+        dry_run = options['dry_run']
+        dry_run_count = options['dry_run_count']
+
+        if from_csv:
+            users = _get_users_from_csv(from_csv)
+            if dry_run_count:
+                sys.stderr.write(f"\n{len(list(users))} users in CSV file '{from_csv}'\n")
+                return
+
+            for domain, request_user, as_username in users:
+                request_user_id = CouchUser.get_by_username(request_user).user_id
+                as_user_id = None
+                if as_username:
+                    as_user_id = CouchUser.get_by_username(as_username).user_id
+                sys.stdout.write(f"{domain},{request_user},{request_user_id},{as_username},{as_user_id}\n")
+                if not dry_run:
+                    clear_formplayer_db_for_user.delay(domain, request_user_id, as_user_id)
+        else:
+            domains = [domain.strip() for domain in domains if domain.strip()]
+            synced_since = datetime.utcnow() - relativedelta(hours=last_synced_hours)
+            not_synced_since = datetime.utcnow() - relativedelta(hours=not_synced_hours) if not_synced_hours else None
+            if dry_run_count:
+                users = list(_get_user_rows(domains, synced_since, not_synced_since, None, limit))
```

Comment:
Nitpick: call `_get_user_rows` before the if instead of having two different calls.